CONTRIBUTING.md

```diff
@@ -0,0 +1,93 @@
+# Contributing Guidelines
+
+## Filing a Bug Report
+
+Simply create a new issue for a bug, and one of us at the VicRoy team will check it out!
+
+In your bug report please include the following 
+* A summary of what the bug is
+* The observed and expected behaviour
+* Steps to reproduce the bug
+* Other contextual information (e.g. version and operating environment)
+* Screenshots would also be great if possible
+
+## Suggesting a New Feature
+
+We have created a roadmap for the project and what features we would like to add as the project progresses. However, we are always open to new ideas and cool 
+suggestions so if you do have a lightbulb moment, feel free to send us an email at vroyale374@gmail.com telling us all about it!
+
+## Setting Up a Pull Request
+
+We have made some custom issue templates to streamline pull requests so that we can quickly see what your contribution addresses. Simply create a pull request
+and select the issue template related to your contribution. The issue template will ask you for some details, please answer them with as much detail as practical
+so that we can get an idea for what you have made and how you did it. From there, one of out team members here at VicRoy will review your pull request
+and give you some constructive feedback. Once it all looks good to go, we would be honoured to merge your contribution!
+
+## How to Set Up Your Environment and Run Tests
+
+??????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????? 
+Dont think this is relevant to us because we dont run tests and i have talked about instalation/prerequisites in readme 
```

Comment:
I reckon just mention how to run the actual android studio application here. Could just link the same video again